src/Patterns/BlockPattern.php

```diff
@@ -43,16 +43,17 @@ public static function get_list(): array
         return [
             BlankPage::class,
             DeepDive::class,
+            GetInformed::class,
             HighLevelTopic::class,
             HighlightedCta::class,
+            Homepage::class,
             Issues::class,
             QuickLinks::class,
             RealityCheck::class,
             SideImageWithTextAndCta::class,
             TakeAction::class,
             PageHeader::class,
             PageHeaderImgLeft::class,
```

Comment:
Assuming you've moved `Homepage::class,` to line 49 to keep it alphabetically ordered, we should also move `PageHeader::class,` and `PageHeaderImgLeft::class,` to line 51.